frontend/src/style.css

```diff
@@ -33,6 +33,10 @@ img {
   height: 35px;
 }
 
+.signatureDiv {
+  width: 501px;
```

Comment:
Ez így nem túl reszponzív :)

Reply:
Van rá másik issue.

Reply:
Értem én, hogy az 501-es HTTP status code a Not Implemented-et jelöli, de ezt nem szoktuk CSS-ben szélességként használni.